freqtrade/freqtradebot.py

```diff
@@ -459,7 +459,9 @@ def check_and_call_adjust_trade_position(self, trade: Trade):
                 return
         else:
             logger.debug("Max adjustment entries is set to unlimited.")
-        current_rate = self.exchange.get_rate(trade.pair, refresh=True, side="buy")
```

Comment:
While not directly on this line - you'll need to change how `max_entry_position_adjustment` is handled/interpreted above, otherwise your "partial sell" will not be called once `max_entry_position_adjustment` is reached.

I'd probably update it to be called all the time - ignoring the buy if `max_entry_position_adjustment` is reached. That'll obviously also have to be updated in the docs.

Reply:
updated

Reply:
If the bot has max rebuy of 3, and then it has used it all and partial sell the last order, should the bot be assumed to have 1 open rebuy slot, or it's not allowed to rebuy again?

Reply:
It is not allowed now. If we want, we can set to buy count - sell count

Reply:
i'm not sure if we need an additional config for this considering that this is mostly a "convenience" functionality - but also something that can easily be handled within the strategy.
We have too many config options barely anyone knows how to use already - no point in making every single question configurable i think.

As such, i think it's more a point on "do we want it this way or that".
As long as the behaviour is documented, i don't see this as a breaking point - it's just "it's like this" ...